### ISSUE TITLE
Subquery not works

### ISSUE THREAD

**Issue by hey-red** [sqlkata/querybuilder#6]
From [docs](http://sqlkata.vivida-apps.com/?sql--sqlsrv#subquery-select)
```C#
var query = new Query().From("Users")
                .Select("Id")
                .Select(q => q.From("Visits").Where("UserId", 10).Count(), "VisitsCount");
```
Expected:
```
SELECT "Id",
  (
    SELECT COUNT(*) FROM "Visits" WHERE "UserId" = 10
  ) AS "VisitsCount"
FROM "Users"
```
Actual:
`SELECT "Id" FROM "Users"`

It seems like `QueryColumn`("column" component) just ignored [here](https://github.com/sqlkata/querybuilder/blob/master/QueryBuilder/Compilers/Compiler.cs#L214) and not handled in `Columnize`

**Comment by ahmad-moussawi** [comment]
Thanks for reporting, fixed now in this commit [c89f9a5](https://github.com/sqlkata/querybuilder/commit/c89f9a55bacd40f5e886111d5b726667f49f1aec)

**Comment by hey-red** [comment]
Thanks!
